src/iso19111/operation/conversion.cpp

```diff
@@ -4126,17 +4126,29 @@ void Conversion::_exportToPROJString(
             "Latitude_Of_Center", common::UnitOfMeasure::DEGREE);
         const double XYPlaneRotation = parameterValueNumeric(
             "XY_Plane_Rotation", common::UnitOfMeasure::DEGREE);
-        if (scaleFactor != 1.0 || azimuth != 0.0 || latitudeOfCenter != 0.0 ||
-            XYPlaneRotation != 0.0) {
+        if (scaleFactor == 1.0 && std::fabs(azimuth - 40.17823482) <= 1e-8 &&
+            std::fabs(longitudeOfCenter - 66.94970198) <= 1e-8 &&
+            std::fabs(latitudeOfCenter - -49.56371678) <= 1e-8 &&
+            std::fabs(XYPlaneRotation - 45) <= 1e-8) {
```

Comment:
If you prefer, you can set the parameters `+lat_c`, `+lon_c`, `rot` and `azi` to use the spilhaus projection.
I added `rot` to make it more generic, and usable with ESRI Adams_ws2